makefile

```diff
@@ -110,7 +114,8 @@ TEST_SET_ENV_VARS := \
 	export CONTACT_BUYING_USER_NOTIFY_TEMPLATE_ID=debug; \
 	export CONTACT_EXPORTING_USER_NOTIFY_TEMPLATE_ID=debug; \
 	export CONTACT_EXPORTING_AGENT_SUBJECT=exporting-subject; \
-	export COMPANIES_HOUSE_API_KEY=debug
+	export COMPANIES_HOUSE_API_KEY=debug; \
+	export FEATURE_EXPORT_JOURNEY_ENABLED=true
```

Comment:
as `FEATURE_EXPORT_JOURNEY_ENABLED` is set in `DEBUG_SET_ENV_VARS`, it's not needed here :)

Reply:
aha. i added this when testing. it must have slipped through